test/spec/features/modeling/ResizeShapeSpec.js

```diff
@@ -12,6 +12,30 @@ import {
 import modelingModule from 'lib/features/modeling';
 import coreModule from 'lib/core';
 
+describe('features/modeling - resize shape with embedded label', function() {
+
+  var diagramXML = require('./ResizeShapeSpec.embeddedLabel.bpmn');
+
+  var testModules = [coreModule, modelingModule];
+
+  beforeEach(bootstrapModeler(diagramXML, { modules: testModules }));
+
+  it('should clear only explicit embedded bounds', inject(function(elementRegistry, modeling) {
+
+    // given
+    var subProcess = elementRegistry.get('Activity_1'), subProcessDi = getDi(subProcess);
```

Comment:
Let's only use one variable assignment per line
```suggestion
    var subProcess = elementRegistry.get('Activity_1'), 
          subProcessDi = getDi(subProcess);
```